README.md

```diff
@@ -69,9 +73,38 @@ The supported metadata keys for omigami are (case insensitive):
 - "inchikey_inchi"
 
 #### Notebooks
-You can find a [tutorial](https://github.com/omigami/romigami/blob/master/notebook/Tutorial.ipynb) notebook in the `/notebooks/` folder.
+You can find a [tutorial](https://github.com/omigami/romigami/blob/master/notebook/spec2vec/tutorial.ipynb) notebook in the `/notebooks/` folder.
+
+And an [R Markdown document](https://github.com/omigami/romigami/blob/master/vignettes/spec2vec/tutorial.Rmd) on the `/vignettes/` folder.
```

Comment:
why do we need this big path with blob/master/ ?

Reply:
We need it to link to the file in the repo.